### PR TITLE
Add express documentation to readme

### LINKED ISSUE #1: Create an Express example in README

### PR DESCRIPTION
**Pull request by KayBeSee**
Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,18 @@ Logs:
 ```
 
 ### Examples
+##### Express
+```var express = require('express');
```
Comment (jackboberg) [inline README.md]: "```" should be on it's on line

File: README.md [hunk 2]
```diff
@@ -27,6 +27,18 @@ Logs:
...
+var app = express();
+var KnockKnock = require('./lib/knock-knock');
```
Comment (jackboberg) [inline README.md]: `require('knock-knock')`

**Comment by jackboberg** [comment]
I like keeping the `hapi` example first, because it's the best!!

**Comment by KayBeSee** [comment]
I knew you'd say that ;)

File: README.md (hunk 2 above)
Comment (franvarney) [inline README.md]: can you change `KnockKnock` to `knockKnock`? :)

Comment (jackboberg) [inline README.md]: Actually, I think you should use `@modulus/standard` style, and use:

`const Express` and `const KnockKnock`

**Comment by franvarney** [comment]
You can go ahead and merge this @KayBeSee :)

**Comment by KayBeSee** [comment]
yes ma'am

**Pull request merged**